src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -495,6 +469,31 @@ transactionPreProcessImpl(
             tx_json[jss::Flags] = tfFullyCanonicalSig;
     }
 
+    {
+        Json::Value err = checkPayment(
```

Comment:
Why `checkPayment` and `checkFee` swap their places?

Reply:
Hm not sure, fixed in 5b42a96fec72ed3246f6073445918aad69efcb36